app/models/course/virtual_classroom_settings.rb

```diff
@@ -42,6 +42,21 @@ def title=(title)
     @settings.title = title
   end
 
+  # Returns the max duration of virtual classrooms component
+  #
+  # @return [String] The custom or default max_duration of virtual classrooms component
+  def max_duration
+    @settings.max_duration || 60
+  end
+
+  # Sets the max duration of virtual classrooms component
+  #
+  # @param [String] max_duration The new max duration
+  def max_duration=(max_duration)
```

Comment:
the 15 minute step is very unlikely to change. So I think it's OK to do it in the view.